lib/fastlane/plugin/revenuecat_internal/actions/trigger_action_in_circle_ci_action.rb

```diff
@@ -0,0 +1,71 @@
+require 'fastlane/action'
+require 'fastlane_core/configuration/config_item'
+require 'fastlane_core/ui/ui'
+require 'rest-client'
+require 'json'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?(:UI)
+
+  module Actions
+    class TriggerActionInCircleCiAction < Action
+      def self.run(params)
+        circle_token = params[:circle_token]
+        UI.user_error!("Please set the CIRCLE_TOKEN environment variable") unless circle_token
+
+        default_branch = Actions.git_branch
```

Comment:
Rather than the `default_branch` this would be the `current_branch` right? As in, it will use whatever branch it's currently in, I think.